test/complexity_test.cc

```diff
@@ -176,6 +176,30 @@ ADD_COMPLEXITY_CASES(n_lg_n_test_name, big_o_n_lg_n_test_name,
 ADD_COMPLEXITY_CASES(n_lg_n_test_name, big_o_n_lg_n_test_name,
                      rms_o_n_lg_n_test_name, lambda_big_o_n_lg_n);
 
+// ========================================================================= //
+// -------- Testing formatting of Complexity with captured args ------------ //
+// ========================================================================= //
+
+#ifdef BENCHMARK_HAS_CXX11
```

Comment:
I'm pretty sure C++11-capable compiler is already required to build the google benchmark, so this isn't needed.